controllers/tekton/pipeline/pipeline_controller.go

```diff
@@ -53,16 +57,54 @@ type Reconciler struct {
 func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	ctx := context.Background()
 
-	klog.Infof("req: %v", req)
+	// for debug
+	// klog.Infof("req: %v", req)
```

Comment:
Please remove these debug statements.

Reply:
Okay, I will push a commit to remove it.